lib/markdown.ts

```diff
@@ -1,20 +1,15 @@
 // General helpers for dealing with markdown files / content.
 
-function removeTrailingNewline(str: string) {
-  return str.replace(/\s+$/, '');
-}
-
 export async function format(str: string, filePath: string): Promise<string> {
   try {
     const { default: prettier } = await import('prettier');
```

Comment:
If you decide to make this function compatible with both Prettier v2 & Prettier v3, this should also change to something like

```js
let prettier = await import('prettier');
prettier = prettier.default ?? prettier;
```

Prettier v3 uses named export.

Reply:
@fisker thanks for the heads up. I'll address this in a separate PR, or I might remove prettier formatting entirely: #185.